.github/workflows/build-test-deploy.yaml

```diff
@@ -29,15 +29,36 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        if: ${{ !github.event.issue.pull_request }}
         with:
           # 0 - all history, it uses in generating part of BUILD in version
           fetch-depth: 0
 
+      - uses: actions/github-script@v3
```

Comment:
Why is it needed? The problem not relevant to the PR.

Reply:
not relevant anymore